### PR TITLE
update dsl to reflect changes to pr job

### PR DESCRIPTION
**Pull request by estute**
@jzoldak  first stage, just update dsl scripts to reflect manually made changes

### CODE REVIEW

**Comment by estute** [comment]
@jz second: add the pr for the microsites pr jobs.  need to suss out whether or not this will clobber the other e2e jobs

File: platform/jobs/edxEndToEndTestsJob.groovy [hunk 1]
```diff
@@ -44,19 +44,41 @@ Map pipelineJob = [ name: 'edx-e2e-tests',
...
               ]
 
-List jobConfigs = [ pipelineJob, prJob ]
+// The microsites-staging-test-pr job is run on every PR to the edx-e2e-tests
+// repo. This is used for development of the tests themselves
+Map micrositesPrJob = [ name: 'microsites-staging-tests-pr',
+                        trigger: 'ghprb',
+                        triggerPhrase: 'jenkins run microsites',
+                        branch: '${ghprbActualCommit}',
+                        description: 'Verify the quality of changes made to the end-to-end tests',
```
Comment (jzoldak) [inline platform/jobs/edxEndToEndTestsJob.groovy]: nit: I think this is a copy/paste typo? 
s/end-to-end/microsites

File: platform/jobs/edxEndToEndTestsJob.groovy [hunk 2]
```diff
@@ -44,19 +44,41 @@ Map pipelineJob = [ name: 'edx-e2e-tests',
...
+                        courseNumber: 'AR-1001',
+                        testScript: 'jenkins/white_label.sh',
+                        context: 'jenkins/microsites',
+                        junitReportPath: '*.xml,reports/*.xml',
+                        junitScale: 5
```
Comment (jzoldak) [inline platform/jobs/edxEndToEndTestsJob.groovy]: what's this do? Is it included for compatibility with the existing job?

Comment (estute) [inline platform/jobs/edxEndToEndTestsJob.groovy]: Exactly. It allows you to define an 'amplification factor' to determine how many failures should fail a build

**Pull request merged**